composer/helpers/v2/composer.lock

```diff
@@ -2643,5 +2780,5 @@
         "ext-json": "*"
     },
     "platform-dev": [],
-    "plugin-api-version": "1.1.0"
+    "plugin-api-version": "2.3.0"
```

Comment:
I did notice this change also crept in with `composer update --lock`. But it should be fine given that we don't use any plugins, and even if we did I suspect we'd want to be on the v2 API given that this is `composer` `v2`.